pylint/pyreverse/printer_factory.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2021 Andreas Finkler <andi.finkler@gmail.com>
+
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+
+from typing import Type
+
+from pylint.pyreverse.dot_printer import DotPrinter
+from pylint.pyreverse.plantuml_printer import PlantUmlPrinter
+from pylint.pyreverse.printer import Printer
+from pylint.pyreverse.vcg_printer import VCGPrinter
+
+filetype_to_printer = {"vcg": VCGPrinter, "puml": PlantUmlPrinter, "dot": DotPrinter}
```

Comment:
I often use ``.plantuml`` personally, I guess adding that won't hurt.
```suggestion
filetype_to_printer = {
    "vcg": VCGPrinter,
    "puml": PlantUmlPrinter,
    "plantuml": PlantUmlPrinter,
    "dot": DotPrinter,
}
```